index.bs

```diff
@@ -1719,6 +1719,45 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+Manifest files are sometimes required. However, they can also add complexity to the web platform,
+which creates developer confusion. The use of different manifests on the web should therefore be kept to a minimum.
+
+Metadata can reside inside a new manifest if that is related to a different function or the fetch timing 
+of that metadata is different. Application metadata should be added to the Web App Manifest or be an extension of it. 
+Parameters for a specific function (like Payment Method Manifest) are a completely different type. Publication Manifest,
+using JSON-LD is another type as well.
+
+Note that in all cases, the naming conventions should be harmonized (see section 2.2)
+ 
+[Note: Due to historical reasons, existing manifests use lowercase, underscore-delimited names. There have been times 
+where it has been beneficial to re-use dictionaries from a manifest in DOM APIs as well, which are camel-cased. 
+One such example is the <a href="https://w3c.github.io/image-resource/">image resource</a>. For this reason, if a 
+key can clearly be expressed as a single word, that is recommended.]
```

Comment:
I don't think image resource is a good example of a manifest. It's IDL, and used for APIs only.

In general I think this parenthetical adds more confusion than it helps. For example "Due to historical reasons, existing manifests use lowercase, underscore-delimited names" doesn't seem right. It's due to following the design principles in section 2.2.

Reply:
True, but it used to be in manifest and it is still used as JSON in the Web App Manifest @marcoscaceres

Reply:
This has been reworded from "due to historical reasons" to "by principle", let me know if that addresses your concerns.

Reply:
I still think referencing image resource here is a bad idea.

Reply:
Agree with @domenic.